### PR TITLE
Text Prop - ellipsizeMode - Clip, Tail

### PR DESCRIPTION
**Pull request by martunjai**
Updated Text Prop - ellipsizeMode - Clip, Tail

### CODE REVIEW

File: ReactSkia/components/RSkComponentText.cpp [hunk 1]
```diff
@@ -87,8 +95,10 @@ void RSkComponentParagraph::OnPaint(SkCanvas *canvas) {
 
             numberOfLines = getNumberOfLines(metrics, props.paragraphAttributes.maximumNumberOfLines, frame.size.height);
             if (numberOfLines){
+                if (strcmp(convertEllipsizeMode(props.paragraphAttributes.ellipsizeMode),"clip")) {
+                    paraStyle.setEllipsis(u"\u2026");
```
Comment (shefayaparwin) [inline ReactSkia/components/RSkComponentText.cpp:91]: What means "u"\u2026" here.

Comment (martunjai) [reply ReactSkia/components/RSkComponentText.cpp:91]: Unicode Character 'HORIZONTAL ELLIPSIS' - (...)

File: ReactSkia/components/RSkComponentText.cpp [hunk 2]
```diff
@@ -117,8 +127,10 @@ void RSkComponentParagraph::OnPaint(SkCanvas *canvas) {
 
             numberOfLines = getNumberOfLines(metrics, props.paragraphAttributes.maximumNumberOfLines, frame.size.height);
             if (numberOfLines){
+                if (strcmp(convertEllipsizeMode(props.paragraphAttributes.ellipsizeMode),"clip")) {
```
Comment (pprabhu-ng) [inline ReactSkia/components/RSkComponentText.cpp]: is it required to convert to string to check for "clip" mode ?
can we directly check for "EllipsizeMode::Clip" ?

**Review by pprabhu-ng — CHANGES_REQUESTED** [review]

**Review by pprabhu-ng — APPROVED** [review]

File: ReactSkia/components/RSkComponentText.cpp (hunk 2 above)
Comment (martunjai) [reply ReactSkia/components/RSkComponentText.cpp]: yes, converting to string to check "clip" mode is not required.

**Review by shefayaparwin — APPROVED** [review]

**Pull request merged**